tests/src/Interop/NativeCallable/NativeCallableTest.csproj

```diff
@@ -37,5 +37,9 @@
   <ItemGroup>
     <Compile Include="NativeCallableTest.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <!-- This is needed to make sure native binary gets installed in the right location -->
+    <ProjectReference Include="CMakeLists.txt" />
```

Comment:
I am not sure I understand how adding a cmake project file reference to the msbuild project helps. I can imagine that the dependency would trigger the build of the managed part if the cmakelists.txt changed, but I don't see how it would ensure the native part installation.